saleor/graphql/schema.graphql

```diff
@@ -928,13 +928,16 @@ type Checkout implements Node & ObjectWithMetadata {
   privateMetadata: [MetadataItem]!
   metadata: [MetadataItem]!
   availableShippingMethods: [ShippingMethod]!
+  availableCollectionPoints: [Warehouse]
```

Comment:
I think this should be `availableCollectionPoints: [Warehouse!]` - I guess we'll never have a list with nulls inside? Also, is this field nullable at all or should it return an empty list when there is no collection points available?

Reply:
Yes, probably even the most restrictive `[Warehouse!]!` could be used here, I would prefer an empty list when there is no collection points available.